master/internal/task/allocation.go

```diff
@@ -400,11 +400,13 @@ func (a *Allocation) ResourcesAllocated(ctx *actor.Context, msg sproto.Resources
 	}
 
 	for cID, r := range a.resources {
-		r.Start(ctx, a.logCtx, spec, sproto.ResourcesRuntimeInfo{
+		if err := r.Start(ctx, a.logCtx, spec, sproto.ResourcesRuntimeInfo{
 			Token:        token,
 			AgentRank:    a.resources[cID].rank,
 			IsMultiAgent: len(a.resources) > 1,
-		})
+		}); err != nil {
+			return fmt.Errorf("starting resources (%v): %w", r, err)
```

Comment:
by the way, is there any preference between `errors.Wrapf` and this?

Reply:
https://determined-ai.slack.com/archives/C6D7MDSQ5/p1648734094223559